### ISSUE TITLE
Define 'uncache' in 'cache_model.py'

### ISSUE THREAD

**Issue by jeff1evesque** [jeff1evesque/machine-learning#1862]
We will define the `uncache` method within `cache_model.py`.

**Comment by jeff1evesque** [comment]
When we attempt to *deserialize* a pickled encoded svm label during the `use_model` session, we get the following error traceback:

```bash
$ python app.py
 * Running on http://127.0.0.1:5000/
 * Restarting with reloader
127.0.0.1 - - [15/Apr/2015 20:13:38] "POST /load-data/ HTTP/1.1" 500 -
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1836, in __call__
    return self.wsgi_app(environ, start_response)
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1820, in wsgi_app
    response = self.make_response(self.handle_exception(e))
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1403, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1817, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1477, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1381, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1475, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/local/lib/python2.7/dist-packages/flask/app.py", line 1461, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/var/www/web_interface/views.py", line 43, in load_data
    elif session_type == 'model_use': response = loader.load_model_use()
  File "/var/www/brain/load_data.py", line 109, in load_model_use
    my_prediction = session.svm_prediction()
  File "/var/www/brain/session/model_use.py", line 41, in svm_prediction
    encoded_labels = Cache_Model().uncache('svm_labels', self.model_id)
  File "/var/www/brain/cache/cache_model.py", line 41, in uncache
    return Serialize_Model(uncached).deserialize()
  File "/var/www/brain/converter/serialize_model.py", line 25, in deserialize
    return pickle.loads(self.model)
UnpicklingError: could not find MARK
```

**Comment by jeff1evesque** [comment]
The above traceback has been resolved in #1892.
